test/parallel/test-repl-envvars.js

```diff
@@ -38,19 +38,25 @@ const tests = [
 function run(test) {
   const env = test.env;
   const expected = test.expected;
+
   const opts = {
     terminal: true,
     input: new stream.Readable({ read() {} }),
     output: new stream.Writable({ write() {} })
   };
 
-  REPL.createInternalRepl(env, opts, function(err, repl) {
+  Object.assign(process.env, env);
```

Comment:
What's the benefit of changing this part of the test and then subsequently deleting these values?

Reply:
I moved the default terminal logic into the generic one. To prevent duplicating the logic, I had to make sure mostly everything relies on `process.env` (it can not be anything else outside from our tests).

If I manipulate the environment once, the next test would fail as the environment variable is already set but I expect it to be something else. So I have to remove the added values after running the test again.